optimade/server/utils.py

```diff
@@ -275,7 +281,33 @@ def retrieve_queryable_properties(schema: dict, queryable_properties: list) -> d
     return properties
 
 
-ENTRY_INFO_SCHEMAS = {
-    "structures": StructureResource.schema,
-    "references": ReferenceResource.schema,
-}
+def get_providers():
+    """Retrieve providers.json from /Materials-Consortia/OPTiMaDe"""
+    import requests
+    from bson.objectid import ObjectId
+
+    mat_consortia_providers = requests.get(
+        "https://raw.githubusercontent.com/Materials-Consortia/OPTiMaDe/develop/providers.json"
+    ).json()
+
+    providers_list = []
+    for provider in mat_consortia_providers.get("data", []):
+        # Remove/skip "exmpl"
+        if provider["id"] == "exmpl":
+            continue
+
+        provider["task_id"] = provider.pop("id")
```

Comment:
I'm still not sure I like the fact that we're using `task_id` all over the place (but can see its illustrative value for `/structures`). Can we just use `id` and remove the alias from `LinkMapper`?

Reply:
For sure! If it doesn't clash with MongoDB I'm all for. Not too familiar with MongoDB, so I am just copying what has been done previously here to not unknowingly break something.